internal/pkg/api/server/templates/service/inputs.go

```diff
@@ -70,7 +72,7 @@ func validateInputs(groups template.StepsGroups, payload []*StepPayload) (out *V
 				allValues = append(allValues, template.InputValue{
 					Id:      input.Id,
 					Value:   value,
-					Skipped: !outInput.Visible,
+					Skipped: !outInput.Visible || !outStep.Configured,
```

Comment:
Bez toho neprejdu testy, `optional-group-not-configured`, `missing-step`, netyka sa to API, ale sablony, a co vrati funkcia `InputIsAvailable`. Tj, nasiel som aj tento bug.